src/index.js

```diff
@@ -18,17 +17,19 @@ const playstationType = {
 
 /** Let's do this */
 (async function() {
-    const choice = await promptly.choose("Which version would you like? (disc or digital)", ["disc", "digital"]);
-    const alarm = await promptly.confirm("Would you like to hear a loud, annoying alarm when we find your PS5? (Y or N)")
+    const choice = process.env.PS5_VERSION ? process.env.PS5_VERSION : await promptly.choose("Which version would you like? (disc or digital)", ["disc", "digital"]);
+    const alarm = process.env.PLAY_ALARM ? true : await promptly.confirm("Would you like to hear a loud, annoying alarm when we find your PS5? (Y or N)")
+
     console.log(`Searching for PlayStation 5 ${choice} edition...`);
+  
     const onSuccess = () => {
         console.log("Found it! Opening queue now...");
-        open(playstationType[choice].url);
+        openURL(playstationType[choice].url);
         if (alarm) {
           playAlarm();
         }
     };
-    checkForPlaystationDirectRedirect(5000, onSuccess, playstationType[choice].id, await puppeteer.launch());
+    checkForPlaystationDirectRedirect(process.env.CHECK_INTERVOL ? process.env.CHECK_INTERVOL : 5000, onSuccess, playstationType[choice].id, await puppeteer.launch());
```

Comment:
I believe the following shorthand method should work:
`process.env?.CHECK_INTERVAL || 5000`

Reply:
Optional chaining and nullish coalesce operators are only available in Node versions 14.0 and above. I originally used the nullish coalesce: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#Browser_compatibility

```
process.env.CHECK_INTERVAL ?? 5000
```

But I found when going to my other machine with Node 12 installed, it errored out. I opted to go for backwards support instead of the helpful syntax. 

I think another option to my the above might be `process.env.CHECK_INTERVAL || 500`. I can/will update to this shorthand.

Just to help - using the optional chaining here `process.env?.CHECK_INTERVAL || 5000`  is sort of moot, the optional chain makes sure the field `env?` exists, and since this is a node provided value that will always exist. The actual value of **CHECK_INTERVAL** is what may be null/undefined.